### ISSUE TITLE
[Adaptive Card iOS - ColumnSet.AllElements.Stretch.Toggle.Json]: Voice Over is conveying the incorrect role as 'Button' for the 'Reply' link. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8438]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.2" control and select it.
5. Navigate to 'Test' control and activate it.
6. Navigate to 'ColumnSet.AllElements.Stretch.Toggle.Jsons' and activate it.
7. The Card will open.
8. Navigate to the 'Reply' link and observe the issue.

**Actual Result:**
While navigating to the 'Reply' link, voice over is conveying the incorrect role as 'Button'. It is announcing as 'Reply Button' and on activating it a new page is getting open.

**Expected Result:**
Screen Reader should convey the correct role information as 'Link' for the 'Reply' and it should announce as 'Reply Link'.

**User Impact:**
It will impact the screen reader user if the screen reader does convey the incorrect role for the element as user will get confused and treat the control as button instead of link.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html 

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/229052650-ce5cdac6-5cfb-4efa-b612-0a0859f667ec.mov

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
closing this issue, the app wasn't built correctly perhaps, only cards under Scenario should be tested, not under Elements. I will make note of this and fix it.

**Comment by vagpt** [comment]
Adding A11yTTValidated tag as per the above comment.
